exp/performance.md

```diff
@@ -0,0 +1,52 @@
+
+Also make sure that there is a stable network connection between the nodes of your cluster, to decrease the latency between nodes.
+
+<a name="startup-time"></a>
+## Container startup time
+
+A very noticeable performance issue is a long wait time when starting an application.
+
+When a user starts an application, Anbox Cloud retrieves the application image and launches a new container for it. By default, Anbox Cloud turns off image compression in LXD when launching a container from an image. This method speeds up the launch of the container (because the image does not need to be uncompressed), but it causes more traffic over the network (because the image is transferred uncompressed). If the network connection between your cluster nodes is rather slow, the overall container startup time might improve by enabling image compression. You can change the default configuration by setting the [images_compression_algorithm](https://charmhub.io/ams-lxd/configure#images_compression_algorithm) configuration on the `ams-lxd` charm.
+
+Another configuration that affects the container startup time is [shiftfs_enabled](https://charmhub.io/ams-lxd/configure#shiftfs_enabled). This configuration is currently disabled by default, because it can cause issues with some Android applications. However, if your applications run fine with `shiftfs_enabled` set, it can considerably improve the container startup time. You should be aware though that support for shiftfs might be dropped in future releases.
+
+You should also check the hooks that you use in your application. If you use any startup hooks (`pre-start` or `post-start`) that take a long time or wait for resources to become available, the container startup is delayed. If you use a `post-stop` hook that prolongs the container shutdown, this might also affect the startup time of new containers (because it might not be possible to start more containers until the existing containers terminate).
+
+<a name="client-devices"></a>
+## Client devices
+
+In addition to optimising the performance of your Anbox Cloud deployment, you must also make sure that the client devices that access it can fully utilise its capabilities.
+
+All client devices that access an Anbox Cloud deployment must be capable of low-latency video decoding. They must also use a compatible version of Android WebView.
+
+Furthermore, the network connection is crucial. When implementing your applications, you must take into account what kind of connection the client devices will usually have (for example, 4G, 5G or WiFi), so that you can optimise the network traffic that your applications require.
+
+Also make sure to optimise the network path from the Anbox Cloud server to the client devices. What exactly that entails depends on your specific use case; often it means that the server should be located close to the clients, but there might be other solutions depending on the network service route.
```

Comment:
Does the last part make sense, or can we be more specific?

Reply:
It's fine to me. Maybe extend a bit like.
For the public cloud, you should choose the closest region to your end users to deploy anbox cloud.
For the bare mental,  installing/deploying servers that are geographically close to your end users.